xplique/plots/metrics.py

```diff
@@ -0,0 +1,158 @@
+    """
+    Make a bar-plot regrouping the score of each method for different metrics
+    Used to compare the different metric scores between methods
+
+    Parameters
+    ----------
+    scores
+        Dictionary of metrics scores
+        The keys are the metric names and the values also dictionaries where
+        The keys are the method names and the values the corresponding method-metric score.
+    sort_metric
+        Name of the metric used to sort the methods (a key of the directory)
+    ascending
+        Specify the sorting direction (need a sort_metric specified)
+    title
+        Title of the graphic
+    filepath
+        Path the file will be saved at.
+        if None, the function will call plt.show()
+    methods_colors
+        Dictionary of colors for each method,
+        the values should be supported by matplotlib as a color (named colors, RGB ...)
+
+    Returns
+    -------
+
+    """
+    # pylint: disable=invalid-name
+
+    metrics = list(scores.keys())
+    # sort values
+    if sort_metric is not None:
+        methods = [
+            k for k, _ in
+            sorted(scores[sort_metric].items(), key=lambda item: item[1], reverse=not ascending)
+        ]
+    else:
+        # get methods from a metric (order do not import)
+        methods = list(scores[list(scores.keys())[0]])
+
+    # set x abscesses for each metric
+    metrics_x = {metrics[i]: i for i in range(len(metrics))}
+
+    # set x delta for each method
+    width = (1-0.2) / len(methods)
+    # -0.4 to get 0.2 between each metric, 0.5 width to center the bars
+    methods_d = {methods[i]: -0.4+(i+0.5)*width for i in range(len(methods))}
+
+    # set methods colors
+    if methods_colors is None:
+        cmap = matplotlib.cm.get_cmap("Set3")
+        methods_colors = {methods[i]: cmap(i/(len(methods)-1)) for i in range(len(methods))}
+
+    _, ax = plt.subplots(figsize=(4+2*len(metrics), 4))
+    handles = {}
+    for metric in metrics:
+        for method in methods:
+            handles[method] = ax.bar(
+                metrics_x[metric] + methods_d[method],
+                scores[metric][method],
+                color=methods_colors[method],
+                align="center",
+                width=width
+            )
+
+    ax.set_title(title)
+    plt.legend(handles.values(), methods, title="attribution methods", bbox_to_anchor=(1.05, 1))
+    ax.set_xlabel("metrics")
+    ax.set_ylabel("score")
+    plt.xticks(list(range(0, len(metrics))))
+    ax.set_xticklabels(metrics)
+    plt.xticks(rotation=0)
+    plt.tight_layout()
+
+    if filepath is not None:
+        plt.savefig(filepath)
+    else:
+        plt.show()
+
+
+def fidelity_curves(
+        detailed_scores: Dict[str, np.ndarray],
+        title: str = "",
+        filepath: Optional[str] = None,
+        methods_colors: Optional[Dict[str, Any]] = None
+):
+    """
+    Plot the evolution curves for Insertion and Deletion metrics
+    Based on their detailed_evaluate() method.
```

Comment:
You do not need a capital B for "Based on". Once the doc generated there will not be any break which will make a weird sentence :)